larq/models.py

```diff
@@ -87,13 +81,16 @@ def summary(model, line_length=None, positions=None, print_fn=None):
         )
 
     header = ("Layer", "Outputs", "# 1-bit", "# 32-bit", "Mem (kB)")
+    metrics_weights = [weight for metric in model.metrics for weight in metric.weights]
 
```

Comment:
Is this where the ignored weights are generated? The code isn't very clear to me

Reply:
It is basically a compact and slightly faster version of:
```python
metrics_weights = []
for metric in model.metrics:
    for weight in metric.weights:
        metrics_weights.append(weight)
```